<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <UInt>0</UInt>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialog</receiver>
   <slot>userIdentify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialog</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>userIdentify()</slot>
  <slot>exit()</slot>
 </slots>
</ui>
